tooling.go

```diff
@@ -24,6 +24,10 @@ func (client *Client) Tooling() *Client {
 	return client
 }
 
+func (client *Client) UnTooling() {
+	client.useToolingAPI = false
+}
+
```

Comment:
If you ever start up a client and use the client.Tooling(), you're never able to re-use that client without the tooling API.  This allows you to `defer client.UnTooling()` so subsequent queries will avoid improperly using the tooling api.